server/storage/backend.go

```diff
@@ -46,7 +46,7 @@ func newBackend(cfg config.ServerConfig, hooks backend.Hooks) backend.Backend {
 	}
 	bcfg.BackendFreelistType = cfg.BackendFreelistType
 	bcfg.Logger = cfg.Logger
-	if cfg.QuotaBackendBytes > 0 && cfg.QuotaBackendBytes != DefaultQuotaBytes {
+	if cfg.QuotaBackendBytes > 0 {
```

Comment:
@ahrtr also, more I think on it, we probably don't want to make any change. The default mmapsize is 10GB is considering 8DB max db limit. So if a `quota backend` is not specified by the user, leaving it to default setting makes sense (i.e. consideration for the default max db.) or otherwise provide the log warning. I believe that might be original thoughts. If you disagree than we can wait for other reviews. Thanks!

Reply:
Thanks @spzala . 

The original thought might not be correct. For example, if a user specifies `--quota-backend-bytes` as DefaultQuotaBytes (2GB), then the mmapsize is still 10GB; if a a user specifies `--quota-backend-bytes` as 1 GB or 3GB, then the mmapsize will be 1GB * 110% or 3 * 110% respectively. It doesn't make sense. 

I think we can either remove `cfg.QuotaBackendBytes != DefaultQuotaBytes ` just like this PR does or change it to something like below,
`if cfg.QuotaBackendBytes > 0 && cfg.QuotaBackendBytes > DefaultQuotaBytes`

I agree this isn't a big deal.  But it would be better to have the fix. What do you think? 

> So if a quota backend is not specified by the user, leaving it to default setting makes sense (i.e. consideration for the default max db.) or otherwise provide the log warning. I believe that might be original thoughts.

Reply:
I changed it to `if cfg.QuotaBackendBytes > 0 && cfg.QuotaBackendBytes > DefaultQuotaBytes`